resources/assets/components/CampaignInbox/index.js

```diff
@@ -15,13 +15,11 @@ class CampaignInbox extends React.Component {
     });
 
     return (
-      <div>
-          <ul>
-            {
-              Object.keys(posts)
-                .map(key => <InboxItem key={key} details={posts[key]}/>)
-            }
-          </ul>
+      <div className="container">
+        {
+          Object.keys(posts)
```

Comment:
This is totally subjective, but this might read nicer with Lodash's `map` function:

```js
import { map } from 'lodash';
// ...

return (
    <div className="container">
        { map(posts, (post, key) => <InboxItem key={key} details={post} /> }
    </div>
);

```